### PR TITLE
Add configurable BCversion variable to config file

### PR DESCRIPTION
**Pull request by laestrada**
Configure the IMI to use a specified version of the tropomi smoothed boundary conditions. Also updates the cannon config.yml paths to use the correct imi BCS and tropomi data paths.

### CODE REVIEW

File: envs/Harvard-Cannon/config.harvard-cannon.yml [hunk 1]
```diff
@@ -145,12 +145,13 @@ RestartDownload: false
...
 ##   ("YYYYMMDD_0000z.nc4" will be appended)
-RestartFilePrefix: "/n/jacob_lab/Lab/seasasfs02/CH4_inversion/InputData/BoundaryConditions/OutputDir_bias_corrected_dk_2/GEOSChem.BoundaryConditions."
-RestartFilePreviewPrefix: "/n/jacob_lab/Lab/seasasfs02/CH4_inversion/InputData/BoundaryConditions/OutputDir_bias_corrected_dk_2/GEOSChem.BoundaryConditions."
+RestartFilePrefix: "/n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi-boundary-conditions/v2023-04/GEOSChem.BoundaryConditions."
+RestartFilePreviewPrefix: "/n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi-boundary-conditions/v2023-04/GEOSChem.BoundaryConditions."
 
 ## Path to GEOS-Chem boundary condition files (for nested grid simulations)
 ## Must put backslash before $ in $YYYY$MM$DD to properly work in sed command
-BCpath: "/n/jacob_lab/Lab/seasasfs02/CH4_inversion/InputData/BoundaryConditions/OutputDir_bias_corrected_dk_2"
+BCpath: "/n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi-boundary-conditions/v2023-04/"
```
Comment (nicholasbalasus) [inline envs/Harvard-Cannon/config.harvard-cannon.yml]: I think /v2023-04/ should be removed here.

Currently, it writes as follows.
```
#==============================================================================
# --- GEOS-Chem boundary condition file ---
#==============================================================================
(((GC_BCs
* BC_ /n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi-boundary-conditions/v2023-04//v2023-04/GEOSChem.BoundaryConditions.$YYYY$MM$DD_0000z.nc4 SpeciesBC_?ADV?  1980-2021/1-12/1-31/* EFY xyz 1 * - 1 1
)))GC_BCs
```

**Comment by laestrada** [comment]
> Can you change the TROPOMI path for cannon as well?
> 
> `/n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi`

@nicholasbalasus Done.

**Review by nicholasbalasus — COMMENTED** [review]
Looks good to merge! I'm running a test on Cannon right now and I'll let you know if anything doesn't work.

File: envs/Harvard-Cannon/config.harvard-cannon.yml (hunk 1 above)
Comment (laestrada) [reply envs/Harvard-Cannon/config.harvard-cannon.yml]: You were 1 commit too far back... try it now

**Pull request merged**